src/people.rs

```diff
@@ -252,7 +250,7 @@ impl Hasher for IndexValueHasher {
     }
 }
 
-type Indexer = dyn Fn(&Context, PersonId) -> (IndexValue, String);
+type PersonCallback<T> = dyn Fn(&Context, PersonId) -> T;
```

Comment:
I guess it's a taste issue, but I probably would just define two functions here.